src/Components/WebAssembly/Authentication.Msal/src/Interop/AuthenticationService.ts

```diff
@@ -56,6 +56,9 @@ class MsalAuthorizeService implements AuthorizeService {
 
     constructor(private readonly _settings: AuthorizeServiceConfiguration) {
         const callbackUrl = location.href;
+        if (this._settings.auth) {
+            this._settings.auth.knownAuthorities = [new URL(this._settings.auth.authority!).hostname]
+        }
```

Comment:
We should make sure this doesn't give AAD any grief

Reply:
Sounds good. I've added this to make my dev loop faster but can remove later if this is not necessary.

Reply:
I think we'll have to expose knownAuthorities in the MsalProviderOptions and maybe do this by default if no authorities are specified, it might be that this ends up clashing with AAD so it is something we must only do in B2C, similar to the ValidateAuthority parameter

Reply:
I've added it as an option to MsalProviderOptions and added fallback logic in the AuthenticationService.

> it might be that this ends up clashing with AAD so it is something we must only do in B2C, similar to the ValidateAuthority parameter

Should we achieve this by setting it to default to an empty list? Not sure what kind of collisions we can expect here...

Reply:
> it might be that this ends up clashing with AAD so it is something we must only do in B2C, similar to the ValidateAuthority parameter

I can imagine AAD having an authority parameter and a knownAuthorities parameter that doesn't match. I'm just saying that we should validate that this doesn't break AAD, just that.